From 678b1f3aa4d0059cbf34ad30b1538c013de05073 Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert@linux-m68k.org>
Date: Fri, 25 Jun 2021 11:52:26 +0200
Subject: [PATCH 09/52] [NOT-FOR-UPSTREAM] clk: Make clk_ignore_unused public

Make clk_ignore_unused public, so the StarFive JH7100 Clock Generator
driver can set the flag.

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 drivers/clk/clk.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/clk.c b/drivers/clk/clk.c
index 65508eb89ec9..8f9a3ece86bf 100644
--- a/drivers/clk/clk.c
+++ b/drivers/clk/clk.c
@@ -1282,7 +1282,7 @@ static void __init clk_disable_unused_subtree(struct clk_core *core)
 		clk_core_disable_unprepare(core->parent);
 }
 
-static bool clk_ignore_unused __initdata;
+bool clk_ignore_unused __initdata;
 static int __init clk_ignore_unused_setup(char *__unused)
 {
 	clk_ignore_unused = true;
-- 
2.30.2

